validphys2/src/validphys/config.py

```diff
@@ -55,6 +55,9 @@
 
 import validphys.scalevariations
 
+import numpy as np
+import pandas as pd
+import pathlib
```

Comment:
Yeah, `isort` would complain otherwise: first section stdlib, second third parties, third section local imports.

P.S.: why do you import from `validphys` instead of:
```py
from . import scalevariations
```
?